doc/api/n-api.md

```diff
@@ -5123,6 +5123,12 @@ invocation. If it is deleted before then, then the finalize callback may never
 be invoked. Therefore, when obtaining a reference a finalize callback is also
 required in order to enable correct disposal of the reference.
 
+_Caution_: Finalizer callbacks may be deferred, leaving a window where the
```

Comment:
Optional suggestion, totally not blocking:

```suggestion
Finalizer callbacks may be deferred, leaving a window where the
```

Reply:
I do feel like this is something that can create bugs (in fact this comes from node-usb/node-usb#441) so I mirrored the `_Caution_:` from the paragraph above, but I understand it's a very specific situation so maybe it's unnecessary. 

Are we trying to move away from "Caution" in general? If so, I'll remove it right away :)

Reply:
I'd say we want to use it very sparingly so I'll probably always suggest removing it. But that doesn't mean my suggestion is a good one. It's a pretty automatic response at this point. So by all means, use your judgment. I'm just making a suggestion to make sure you really mean it.

Reply:
I see, thanks. In that case I'll remove it